Python/README.md

```diff
@@ -137,6 +137,8 @@ _add list here_
 - [Merging Sorted arrays](other/merge_sorted_arrays.py)
```

Comment:
@rutujadhanawade Just add `- [Maximum Square Matrix with all 1's](other/Maximum_Square_Matrix.py)` in README in alphabetical order. Do not edit other links.

Reply:
@iamrajiv done :+1: